src/Admin/Settings/Page.php

```diff
@@ -228,6 +234,7 @@ public function enqueue_assets( $hook_suffix ) {
 			return;
 		}
 
+		wp_enqueue_style( 'rocketcdn-settings', $this->assets_baseurl . 'css/settings.css', [], '1.0' );
```

Comment:
The version param here probably not be hard-coded?

Reply:
Should we tie the assets version to the plugin version, or each asset have its own version number, only changing when the specific asset changed?

Reply:
I think the asset should track the plugin version. If the asset is updated there will need to be a plugin version update as well, and we can always get from version control what the asset looked like with any release.